src/main/kotlin/de/achimonline/ansible_lint/settings/AnsibleLintConfigurable.kt

```diff
@@ -44,6 +45,15 @@ class AnsibleLintConfigurable : BoundConfigurable(message("settings.display.name
                         executeTest()
                     }.component
                 }
+                row {
+                    val pathField = textField()
+                        .label(message("settings.group.executable.path"), )
+                        .comment(message("settings.group.executable.path.comment"))
+                        .bindText(settings::path)
+                    button(message("settings.group.executable.path.button.default")) {
```

Comment:
This button might be misleading.
It will **update** the "path setting" (and permanently store it) based on the system's current settings; which is a manual/deliberate process in this case.

Not sure if statically storing the full/current `PATH` environment in the plugin's settings is a good idea.

Are more dynamic/automatic handling would be preferred here.